src/Categories/Diagram/Coequalizer/Properties.agda

```diff
@@ -63,8 +63,325 @@ regular-is-coeq-kp {A} {B} f record { C = D ; h = h ; g = g ; coequalizer = coeq
+
+module MapBetweenCoequalizers where
+
+  ⇒coequalize : {A₁ B₁ A₂ B₂ : Obj}
+              → {f₁ g₁ : A₁ ⇒ B₁} → {f₂ g₂ : A₂ ⇒ B₂}
+              → (α : A₁ ⇒ A₂) → (β : B₁ ⇒ B₂)
+              → CommutativeSquare α f₁ f₂ β                -- f₂ ∘ α ≈ β ∘ f₁
+              → CommutativeSquare α g₁ g₂ β                -- g₂ ∘ α ≈ β ∘ g₁
+              → (coeq₂ : Coequalizer f₂ g₂)
+              → (Coequalizer.arr coeq₂ ∘ β) ∘ f₁ ≈ (Coequalizer.arr coeq₂ ∘ β) ∘ g₁
+  ⇒coequalize {A₁} {B₁} {A₂} {B₂} {f₁} {g₁} {f₂} {g₂} α β sq₁ sq₂ coeq₂ = begin
+    (Coequalizer.arr coeq₂ ∘ β) ∘ f₁ ≈⟨ extendˡ (⟺ sq₁) ⟩
+    (Coequalizer.arr coeq₂ ∘ f₂) ∘ α ≈⟨ equality₂ ⟩∘⟨refl ⟩
+    (Coequalizer.arr coeq₂ ∘ g₂) ∘ α ≈⟨ extendˡ sq₂ ⟩
+    (Coequalizer.arr coeq₂ ∘ β) ∘ g₁ ∎
+    where
+      open HomReasoning
+      open MR 𝒞
+      open Coequalizer coeq₂
+      open IsCoequalizer isCoequalizer renaming (equality to equality₂)
+
+  ⇒MapBetweenCoeq : {A₁ B₁ A₂ B₂ : Obj}
+                  → {f₁ g₁ : A₁ ⇒ B₁}
+                  → {f₂ g₂ : A₂ ⇒ B₂}
+                  → (α : A₁ ⇒ A₂)
+                  → (β : B₁ ⇒ B₂)
+                  → CommutativeSquare α f₁ f₂ β                -- f₂ ∘ α ≈ β ∘ f₁
+                  → CommutativeSquare α g₁ g₂ β                -- g₂ ∘ α ≈ β ∘ g₁
+                  → (coeq₁ : Coequalizer f₁ g₁)
+                  → (coeq₂ : Coequalizer f₂ g₂)
+                  → Coequalizer.obj coeq₁ ⇒ Coequalizer.obj coeq₂
+  ⇒MapBetweenCoeq α β sq₁ sq₂ coeq₁ coeq₂ = coequalize₁ (⇒coequalize α β sq₁ sq₂ coeq₂)
+    where
+      open Coequalizer coeq₁ renaming (isCoequalizer to isCoequalizer₁)
+      open IsCoequalizer isCoequalizer₁ renaming (coequalize to coequalize₁)
+      open HomReasoning
+
+  ⇒MapBetweenCoeqSq : {A₁ B₁ A₂ B₂ : Obj}
+                  → {f₁ g₁ : A₁ ⇒ B₁}
+                  → {f₂ g₂ : A₂ ⇒ B₂}
+                  → (α : A₁ ⇒ A₂)
+                  → (β : B₁ ⇒ B₂)
+                  → (sq₁ : CommutativeSquare α f₁ f₂ β)               -- f₂ ∘ α ≈ β ∘ f₁
+                  → (sq₂ : CommutativeSquare α g₁ g₂ β)               -- g₂ ∘ α ≈ β ∘ g₁
+                  → (coeq₁ : Coequalizer f₁ g₁)
+                  → (coeq₂ : Coequalizer f₂ g₂)
+                  → CommutativeSquare
+                      β (Coequalizer.arr coeq₁)
+                      (Coequalizer.arr coeq₂) (⇒MapBetweenCoeq α β sq₁ sq₂ coeq₁ coeq₂)
+  ⇒MapBetweenCoeqSq α β sq₁ sq₂ coeq₁ coeq₂ = universal₁
+    where
+      open Coequalizer coeq₁ renaming (isCoequalizer to isCoequalizer₁)
+      open IsCoequalizer isCoequalizer₁ renaming (universal to universal₁)
+
+open MapBetweenCoequalizers public
+
+CoeqOfIsomorphicDiagram : {A B : Obj} {f g : A ⇒ B} (coeq : Coequalizer f g )
+                        → {A' B' : Obj}
+                        → (a : A ≅ A') (b : B ≅ B')
+                        → Coequalizer (_≅_.from b ∘ f ∘ _≅_.to a) (_≅_.from b ∘ g ∘ _≅_.to a)
+CoeqOfIsomorphicDiagram {A} {B} {f} {g} coeq {A'} {B'} a b = record
```

Comment:
please remove the implicits that are not actually used below (and everywhere in this PR).